src/parser/content-evaluator-module.ts

```diff
@@ -74,6 +74,19 @@ export class ContentEvaluatorModule implements Module {
     return result;
   }
 
+  _getRewardForComment(comment: GithubCommentScore | undefined, relevance: number) {
```

Comment:
I would switch these args around and then make it 

comment?:

Instead of writing undefined as the type

Reply:
This is semantically different. Having `undefined` on the right side requires the argument, which can eventually be undefined. Using `?` means the argument can be omitted. I forgot the whole context, will check if that's possible, but having a relevance without comment doesn't make sense to me.

Reply:
Then remove `| undefined`

It's logically the same